core-v1/src/main/scala/dsl/Wrappable.scala

```diff
@@ -0,0 +1,24 @@
+package com.rallyhealth.vapors.v1
+
+package dsl
+
+import shapeless.unexpected
+
+import scala.annotation.implicitAmbiguous
+
+trait Wrappable[A]
+
+object Wrappable extends Wrappable[Any] {
+
+  implicit def id[A]: Wrappable[A] = Wrappable.asInstanceOf[Wrappable[A]]
+
+  @implicitAmbiguous(
+    """Can't use the .get operation to select a field of type ${C}[${A}].
+
+This is because there is no Functor[${C}] definition. You can use .getAs[Seq] to convert the value to a Seq before mapping over the results.""",
+  )
+  implicit def setIsNotWrappable1[C[a] <: Set[a], A]: Wrappable[C[A]] = unexpected
+  implicit def setIsNotWrappable2[C[a] <: Set[a], A]: Wrappable[C[A]] = unexpected
+
+  // TODO: Other common non-Functor output types?
```

Comment:
Maybe I should add `Array`? It is probably rare, but common enough to warrant a special compiler error message.